cookbooks/cdo-repository/recipes/default.rb

```diff
@@ -61,3 +61,27 @@
     not_if {File.exist? worktree_path}
   end
 end
+
+# Create sparse checkout of staging content directories on production-daemon
+# for the purpose of content seeding.
+if node.name == 'production-daemon'
```

Comment:
Eventually, every installation of our application except levelbuilder and test would likely be subscribing to the curriculum content committed to our main branch (which we call `staging`), and node names are a little brittle.

Maybe we should switch this login to test that we're running on a daemon  AND that the environment type is `production`.  I think we could start with this and then expand the array later to include `adhoc`:

``` ruby
(node['cdo-apps']['daemon'] && %w[production].include?(node.chef_environment))
```

Reply:
Done.